src/span_processor.ts

```diff
@@ -21,7 +21,7 @@ export class SpanProcessor implements OpenTelemetrySpanProcessor {
   onStart(_span: Span, _parentContext: Context): void {}
 
   onEnd(span: ReadableSpan): void {
-    this.client.extension.importOpenTelemetrySpan(
+    const opentelemetrySpan = this.client.extension.createOpenTelemetrySpan(
```

Comment:
```suggestion
    const opentTelemetrySpan = this.client.extension.createOpenTelemetrySpan(
```

I'm not sure if we have a convention for this naming, but to keep up with the OpenTelemetry convention I prefer casing the T from Telemetry